agents/python/configurator/lib_handling.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import importlib
+from importlib import metadata as md
+
+def reorder_python_path():
+    paths_to_move = [path for path in sys.path if path.startswith('/var/odigos/python')]
+    
+    for path in paths_to_move:
+        sys.path.remove(path)
+        sys.path.append(path)
+    
+    
+def reload_distro_modules() -> None:
+    # Reload distro modules, as they may have been imported before the path was reordered.
+    # Add any new distro modules to this list.
+    needed_modules = [
+        'google.protobuf',
+        'requests',
+        'charset_normalizer',
+        'certifi',
+        'asgiref'
+        'idna',
+        'deprecated',
+        'importlib_metadata',
+        'packaging',
+        'psutil',
+        'zipp',
+        'urllib3',
+        'uuid_extensions.uuid7',
+        'typing_extensions',
```

Comment:
Are these taken from `requirements.txt`? If so, maybe it is better to read them from the file or pass them as an argument

Reply:
Hey Ron,
This file is not exists in the runtime, also it is containing packages names that sometimes are different from the module name that i'm reloading.
**e.g** -> protobuf (`pip install protobuf`) but the module name is `google.protobuf`.
I thought about reloading all modules available in sys.modules but it can potentially take time